javascript/javascript2/week1/findthedanishletters.js

```diff
@@ -0,0 +1,17 @@
+function findDanishLetter(danishString) {
+    const dasnishLetters = ['æ', 'ø', 'å'];
```

Comment:
Watch out for typos in the names of the variables! Sometimes they can make you have a real hard time trying to figure out what's wrong🤯. **dasnishLetters > danishLetters**